src/services/httpService.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+export default {
```

Comment:
why do this ?
if anything it is much longer to do import for the services than the axios package
`import http from "../../services/httpService";`
`import axios from "axios";`

Reply:
i dont understand.
you want me to import axios in BusinessesList component?
if yes then you coupled axios with this component and i'm sure with other components in the future.
i'll do what you asked me at the end , its just a suggestion.

Reply:
it's not worth it to put axios functions in different files, unless you want to manipulate or change the `get` function of axios. so if you are not changing anything (like adding headers to the get request or any axios settings) no need to have it in a different file

Reply:
ok